pkg/drivers/kic/oci/oci.go

```diff
@@ -323,3 +358,22 @@ func listContainersByLabel(ociBinary string, label string) ([]string, error) {
 	}
 	return lines, err
 }
+
+// PointToHostDockerDaemon will unset env variables that point to docker inside minikube
+// to make sure it points to the docker daemon installed by user.
+func PointToHostDockerDaemon() error {
+	p := os.Getenv(constants.MinikubeActiveDockerdEnv)
+	if p != "" {
+		glog.Infof("shell is pointing to docker inside minikube. will unset to use host")
+	}
+
+	for i := range constants.DockerDaemonEnvs {
+		e := constants.DockerDaemonEnvs[i]
+		err := os.Setenv(e, "")
```

Comment:
Does `minikube start` work with this PR?

Instead of manipulating a global variable, I think this would be cleaner if it returned a sanitized environment map.

The new environment can then be passed into `cmd.Env`

One caveat  I see however is that this may not be directly be applicable to go-containerregistry calls in start.go. Is this a workable idea?

Reply:
Yeah the start and status and stop and delete works.

Reply:
I totally agree that it is much much better to pass it as cmd.Env and i will send a follow-up pr to improve it after release

Reply:
On a second thought though. If we don't set it globally how can we make go container register use host docker?

I think in no case minikube needs to talk to the docker-env inside minikube ( for that we run our command inside the node)